CoreMapView/src/au/gov/asd/tac/constellation/views/mapview2/MapView.java

```diff
@@ -1237,16 +1215,16 @@ private boolean selectedMarkersInView(final boolean allMarkers) {
                 final double x = (m.getX() - 100);
                 final double y = (m.getY() + 80);
 
-                Rectangle r = new Rectangle();
+                final Rectangle r = new Rectangle();
                 r.setWidth(10);
                 r.setHeight(17.5);
                 r.setX(x);
                 r.setY(y);
 
-                if (!(parent.getViewPortRectangle().contains(mapStackPane.localToParent(x, y)) && parent.getViewPortRectangle().contains(mapStackPane.localToParent(x + r.getWidth(), y)) && parent.getViewPortRectangle().contains(mapStackPane.localToParent(x, y + r.getHeight())) && parent.getViewPortRectangle().contains(mapStackPane.localToParent(x + r.getWidth(), y + r.getHeight())))) {
-                    return false;
-                }
-
+                return (!parent.getViewPortRectangle().contains(mapStackPane.localToParent(x, y))
+                        || !parent.getViewPortRectangle().contains(mapStackPane.localToParent(x + r.getWidth(), y))
+                        || !parent.getViewPortRectangle().contains(mapStackPane.localToParent(x, y + r.getHeight()))
+                        || !parent.getViewPortRectangle().contains(mapStackPane.localToParent(x + r.getWidth(), y + r.getHeight())));
```

Comment:
this seems like it's now doing the opposite (the boolean statement is equivalent to old one, but now it returns true instead of false)